documentation/src/docs/recovery-procedures/table-segment-recovery.md

```diff
@@ -0,0 +1,186 @@
+
+    ```
+      data-recovery tableSegment-recovery <directory_where_you_copied to in step 3> <Table Segment name> <directory where you want to copy the output chunks to>
+
+      Ex:-
+        data-recovery tableSegment-recovery /foo/bar completedTransactionsBatch-0 /bar/foo/bar
+    ```
+
+5) Copy these generated Table Segment Attribute Index chunks back to the Tier-2 directory we identified in step 2.
+
+6) Edit the metadata of the Segments to reflect these chunks.
+
+    a)  For example, lets say the chunk generated by the command in the folder is :-
+ 
+               13003178 Jun  3 02:55 completedTransactionsBatch-0$attributes.index.E-1-O-0.09cb6598-da98-43ad-8d9c-44ca6d523c4b
+
+    b)  Copy this chunk to the Tier-2 directory i.e `/mnt/tier2/_system/_tables/`
+
+
+    c)  Remove the older set of chunks. That is from the above "ls" listed files above, one would remove:-
+
+           -rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101208573.10e6a6e8-8665-4e57-bfe4-085eee5cf46b'
+           -rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101337340.a2e5cf79-914e-4a01-9c35-b390a6f61f4a'
+           -rw-r--r--  1 osboxes osboxes  19876  Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-102496243.b032e528-589b-421b-8977-178d72128dcf'
+          
+
+    d) Perform a get on the Attribute Index Segment in the CLI:
+
+       table-segment get _system/containers/storage_metadata_<owning Container ID here> <segmentName>$attributes.index <SS_Pod_IP_OWNING_THE_CONTAINER>
+
+
+    e) Lets say one gets the below details by performing "get" above.
+
+       
+          > table-segment get _system/containers/storage_metadata_3 _system/_tables/completedTransactionsBatch-0$attributes.index 127.0.1.1
+		For the given key: _system/_tables/completedTransactionsBatch-0$attributes.index
+		SLTS metadata info: 
+		key = _system/_tables/completedTransactionsBatch-0$attributes.index;
+		version = 1654241463251;
+		metadataType = SegmentMetadata;
+		name = _system/_tables/completedTransactionsBatch-0$attributes.index;
+		length = 13488706;
+		chunkCount = 3;
+		startOffset = 13135328;
+		status = 1;
+		maxRollingLength = 128767;
+		firstChunk = _system/_tables/completedTransactionsBatch-0$attributes.index.E-5-O-13131945.29bd2254-2e77-4338-a41a-f9c2af07b913;
+		lastChunk = _system/_tables/completedTransactionsBatch-0$attributes.index.E-5-O-13389479.5779f8d5-89ad-4bc6-87fe-3f99480a7271;
+		lastModified = 0;
+		firstChunkStartOffset = 13131945;
+		lastChunkStartOffset = 13389479;
+		ownerEpoch = 5;
+
+       
+        One would edit the following fields to update the metadata to reflect the chunk properties we have in step 5a).
+
+              
+                - Increment the version.
+                - Update the length to reflect the cumulative length of chunk(s). In the case above we would update it to 13003178.
```

Comment:
`would update it to 13003178` -> why? how a users knows the value to set here?

Reply:
Have added more information.